source/module_hamilt_general/module_xc/xc_functional_vxc.cpp

```diff
@@ -164,6 +176,14 @@ std::tuple<double,double,ModuleBase::matrix> XC_Functional::v_xc(
     }//end if
     // energy terms, local-density contributions
 
+    if (PARAM.inp.nspin == 4 && PARAM.inp.multicolin)
+    { // noncollinear case added by Xiaoyu Zhang, Peking University, 2024.10.02.  multicollinear method for lda Since NCLibxc needs libxc, this part codes will not be used.
+
+        std::cerr << "Error: Multi-collinear approach does not support running without Libxc." << std::endl;
```

Comment:
please use `WARNING_QUIT` function to quit the program.

Reply:
I am not familiar with the function in ABACUS. If you think that it is necessary, you can conduct this small modification so that it fits your requirements.